modules/module-extras.php

```diff
@@ -82,3 +82,25 @@ function jetpack_widgets_add_suffix( $widget_name ) {
 	);
 }
 add_filter( 'jetpack_widget_name', 'jetpack_widgets_add_suffix' );
+
+add_action( 'blog_privacy_selector', 'priv_notice_privacy_selector' );
+
+/**
+ * Displays the option to enable privacy in Settings.
+ */
+function priv_notice_privacy_selector() {
```

Comment:
I would recommend prefixing this function with `jetpack_` to avoid any potential conflicts with other plugins.

Reply:
Implemented in [aaf7862](https://github.com/Automattic/jetpack/pull/10333/commits/aaf78620ceb4629a874763923b9c282fd8c7786f)